From ff17f6fdcc4638b72727fb478ed62b871f32d45f Mon Sep 17 00:00:00 2001
From: ingframin <framin2@gmail.com>
Date: Mon, 9 Jul 2018 21:23:38 +0200
Subject: [PATCH] fixed

---
 agent_node.py | 7 +++----
 main.py       | 5 +++--
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/agent_node.py b/agent_node.py
index 2c8fcda..8d37e25 100644
--- a/agent_node.py
+++ b/agent_node.py
@@ -1,7 +1,7 @@
 from random import randint
 from random import shuffle
 
-<<<<<<< HEAD
+
 #Not active yet, started refactoring
 class AccessPointComponent:
 
@@ -90,10 +90,9 @@ class StationComponent:
         self.candidates = []
 ########################################################################################################
 
-class AgentNode:
-=======
+
 class Agent:
->>>>>>> 8a406e8f8caea84f94777d5aac1dbe0c4b642fd1
+
     '''Base class for node agents'''
     def __init__(self,address=0, x=0, y=0):
         self.address = address
diff --git a/main.py b/main.py
index 86903e1..20dedc0 100644
--- a/main.py
+++ b/main.py
@@ -96,8 +96,9 @@ while running:
             print('pings list='+str(agents_table[n.id].pings), file=f)
         print('input = '+str(agents_table[n.id].message_in), file=f)
         print('output = '+str(agents_table[n.id].message_out), file=f)
-    
-    for n in nodes:
+
+    nds = list(nodes)
+    for n in nds:
         if agents_table[n.id].battery <= 0:
             print(agents_table[n.id].battery)
             input()
-- 
2.10.1.windows.1

